README.md

```diff
@@ -28,13 +27,298 @@ pip install -U pyinstaller-hooks-contrib
 
 Either `a-package` works fine without a hook, or no-one has contributed hooks.
 If you'd like to add a hook, or view information about hooks,
-please see [the wiki](https://github.com/pyinstaller/pyinstaller-hooks-contrib/wiki).
+please see below.
 
 
 ## I want to help!
 
-Please start by providing pull requests and helping solve issues.
+If you've got a hook you want to share then great!
+The rest of this page will walk you through the process of contributing a hook.
+If you've been here before then you may want to skip to the [summary checklist](#summary)
+
+**Please provide one hook per pull request! If you have more than one then submit them in separate pull requests.**
+
+
+### Setup
+
+[Fork this repo](https://github.com/pyinstaller/pyinstaller-hooks-contrib/fork) if you haven't already done so.
+(If you have a fork already but its old, click the **Fetch upstream** button on your fork's homepage.)
+Clone and `cd` inside your fork by running the following (replacing `bob-the-barnacle` with your github username):
+
+```
+git clone https://github.com/bob-the-barnacle/pyinstaller-hoooks-contrib.git
+cd pyinstaller-hooks-contrib
+```
+
+Create a new branch for you changes (replacing `foo` with the name of the package):
+You can name this branch whatever you like.
+
+```
+git checkout -b hook-for-foo
+```
+
+If you wish to create a virtual environment then do it now before proceeding to the next step.
+
+Install this repo in editable mode.
+This will overwrite your current installation.
+(Not that you can reverse this with `pip install --force-reinstall pyinstaller-hooks-contrib`).
+
+```
+pip install -e .
+pip install -r requirements-test.txt
+pip install flake8
+```
+
+Note that on macOS and Linux, `pip` may by called `pip3`.
+If you normally use `pip3` and `python3` then use `pip3` here too.
+You may skip the 2<sup>nd</sup> line if you have no intention of providing tests (but please do provide tests!).
+
+
+### Add the hook
+
+Hooks live in the [src/_pyinstaller_hooks_contrib/hooks/stdhooks/](../master/src/_pyinstaller_hooks_contrib/hooks/stdhooks/) directory.
+Simply copy your hook into there.
```

Comment:
Would it be worth giving guidance on how to get started creating a hook if you have absolutely no idea? Like:
```
A common technique for creating a hook is to get the hook working as quickly as possible, then narrow down the minimal logic to keep it working:

- [ ] Write a test, ie. test_foo(). If you're not sure how to write a test try _import foo_, or a minimal reproduceable example of the code that caused you to determine the hook is needed
- [ ] Run the test and ensure it fails: _pytest -k test_foo_ 
- [ ] Create a new hook file and populate it with:
     _from PyInstaller.utils.hooks import collect_all_
    _datas, binaries, hiddenimports = collect_all("foo")_
- [ ] Rerun your test
    If it fails: Additional work will be needed. If you require assistance please feel free to submit a draft and ask for help.
- [ ] If the test passes, replace your existing hook with:
    _datas = copy_metadata('foo')
    binaries = collect_dynamic_libs('foo')
    hiddenimports = collect_submodules('foo')_
- [ ] If the tests still passes, determine if any of these lines can be removed to create a minimal hook
```

Reply:
I'd quite like something like that on the [page about hooks in the docs](https://pyinstaller.readthedocs.io/en/latest/hooks.html) rather than all the drivel that's on that page currently. Then this readme can just reference it.